cdhweb/pages/templatetags/core_tags.py

```diff
@@ -13,18 +13,23 @@ def site_footer(context):
     """
     Returns the site footer data.
     """
-    # Get footer columns
-    footer_columns = []
+    # Get footer items
     footers = Footer.objects.prefetch_related(
-        "footer_columns", "footer_columns__column_items", "imprint_links"
+        "contact_links", "physical_address", "useful_links", "imprint_links"
     )
```

Comment:
If you make this just
```
footer = Footer.objects.prefetch_related(...).first()
```
then your condition becomes
```
if footer:
```
and all your lookups don't need the intervening `.first()`